frontend/src/components/TradeBox/EncryptedChat/index.tsx

```diff
@@ -47,10 +47,10 @@ const EncryptedChat: React.FC<Props> = ({ orderId, userNick }: Props): JSX.Eleme
   const messageEndRef = useRef<HTMLInputElement>(null);
   const [connected, setConnected] = useState<boolean>(false);
   const [peerConnected, setPeerConnected] = useState<boolean>(false);
-  const [ownPubKey, setOwnPubKey] = useState<string>();
-  const [ownEncPrivKey, setOwnEncPrivKey] = useState<string>();
+  const [ownPubKey] = useState<string>(systemClient.getCookie('pub_key') || '');
+  const [ownEncPrivKey] = useState<string>(systemClient.getCookie('enc_priv_key') || '');
```

Comment:
Aren't these keys malformed? OpenPGP wants "Armored ACII" with linebreaks, but cookies can only be a string. So if I remember well, `\n` is replaced with `\\` when saving to cookies. Maybe these two lines also need the `.split('\\').join('\n')` ?